src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-file-panel.component.ts

```
@@ -2,57 +1,66 @@
+    );
 
-        this.addedLineCount = this.diffEntries
-            .filter((entry) => entry && entry.filePath && entry.startLine && entry.lineCount)
-            .flatMap((entry) => {
-                return this.solutionFileContent?.split('\n').slice(entry.startLine! - 1, entry.startLine! + entry.lineCount! - 1);
-            })
-            .filter((line) => line && line.trim().length !== 0).length;
+    readonly removedLineCount = computed(
+        () =>
+            this.diffEntries()
+                .filter((entry) => entry && entry.previousFilePath && entry.previousStartLine && entry.previousLineCount)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)